### PR TITLE
mgmt-1145 openshift version

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: deploy/assisted-service-configmap.yaml [hunk 1]
```diff
@@ -9,9 +9,16 @@ data:
   SERVICE_BASE_URL: REPLACE_BASE_URL
   NAMESPACE: REPLACE_NAMESPACE
   BASE_DNS_DOMAINS: REPLACE_DOMAINS # example: name1:id1/provider1,name2:id2/provider2
-  OPENSHIFT_INSTALL_RELEASE_IMAGE: "quay.io/ocpmetal/ocp-release:4.6.0-0.nightly-2020-08-31-220837"
   CREATE_S3_BUCKET: "true"
   ENABLE_AUTH: REPLACE_AUTH_ENABLED_FLAG
   JWKS_URL: REPLACE_JWKS_URL # example https://example.com/.well-known/jwks.json
   OCM_BASE_URL: REPLACE_OCM_BASE_URL
   SKIP_CERT_VERIFICATION: "true"
+  OPENSHIFT_VERSIONS: |-
```
Comment (eranco74) [inline deploy/assisted-service-configmap.yaml:17]: :1st_place_medal:

File: deploy/assisted-service-configmap.yaml [hunk 2]
```diff
@@ -9,9 +9,16 @@ data:
...
+    {
+      "4.6": {
+        "rhcos" :"https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/46.82.202008260918-0/x86_64/rhcos-46.82.202008260918-0-metal.x86_64.raw.gz",
```
Comment (eranco74) [inline deploy/assisted-service-configmap.yaml:20]: Where is this (rhcos) used?

File: deploy/assisted-service-configmap.yaml [hunk 3]
```diff
@@ -9,9 +9,16 @@ data:
...
+        "mco": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dc1a34f55c712b2b9c5e5a14dd85e67cbdae11fd147046ac2fef9eaf179ab221",
```
Comment (eranco74) [inline deploy/assisted-service-configmap.yaml:21]: Where is this (mco) used?

**Comment by yevgeny-shnaidman** [comment]
Igal, isn't installcmd command needs to be changed also? You are going to send rchos and mco, no?

File: deploy/assisted-service-configmap.yaml (hunk 2 above)
Comment (tsorya) [reply deploy/assisted-service-configmap.yaml:20]: in installer?

File: deploy/assisted-service-configmap.yaml (hunk 3 above)
Comment (tsorya) [reply deploy/assisted-service-configmap.yaml:21]: same, it will be passed to installer

File: internal/common/tar_actions.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
+package common
```
Comment (tsorya) [inline internal/common/tar_actions.go:1]: this file was just moved from common.py to tar_actions. No need to review it

File: deploy/assisted-service-configmap.yaml (hunk 2 above)
Comment (tsorya) [reply deploy/assisted-service-configmap.yaml:20]: i will provide it to installer and delete map that we have there

**Comment by tsorya** [comment]
> Igal, isn't installcmd command needs to be changed also? You are going to send rchos and mco, no?

@yevgeny-shnaidman i will do it as a next step. If this one is ok i will change installer and installercmd accordingly

**Comment by YuviGold** [comment]
OCP multiversion is supported since https://github.com/openshift/assisted-service/pull/805
/close

**Pull request closed without merging**